examples/herding_approximate_gram_matrix.py

```diff
@@ -54,13 +53,13 @@
 # pylint: disable=too-many-locals
 def main(out_path: Path | None = None) -> tuple[float, float]:
     """
-    Run the tabular data herding example with an approximate kernel matrix row sum mean.
+    Run the tabular data herding example with an approximate kernel Gramian row-mean.
```

Comment:
You don't use the hyphen elsewhere so probably remove to keep consistent, same for 60